packages/flame/lib/src/events/messages/tap_up_event.dart

```diff
@@ -36,4 +36,9 @@ class TapUpEvent extends PositionEvent {
       ),
     );
   }
+
+  @override
+  String toString() =>
+      'TapUpEvent(pointerId;$pointerId, deviceKind:$deviceKind, '
```

Comment:
```suggestion
      '$TapUpEvent(pointerId;$pointerId, deviceKind:$deviceKind, '
```
Will help in keeping class name in this string in-sync with the actual class name in future as well.